stable/datadog/templates/container-agent.yaml

```
@@ -81,13 +81,13 @@
     - name: DD_KUBERNETES_POD_ANNOTATIONS_AS_TAGS
       value: '{{ toJson .Values.datadog.podAnnotationsAsTags }}'
     {{- end }}
-    {{- if .Values.datadog.logsEnabled }}
+    {{- if or .Values.datadog.logs.enabled .Values.datadog.logsEnabled }}
```

This change implies that the default for `DD_LOGS_ENABLED=false` will never change, is that safe to assume?

---

right do you think if I do
```suggestion
    - name: DD_LOGS_ENABLED
      value: {{  (default false (or .Values.datadog.logs.enabled .Values.datadog.logsEnabled)) | quote}}
```

---

right, what do you think if I do:

```suggestion
    - name: DD_LOGS_ENABLED
      value: {{  (default false (or .Values.datadog.logs.enabled .Values.datadog.logsEnabled)) | quote}}
```

---

right, what do you think if I do:

```suggestion
    - name: DD_LOGS_ENABLED
      value: {{  (default false (or .Values.datadog.logs.enabled .Values.datadog.logsEnabled)) | quote}}
```

---

Sure this works!